File: Easy/Temperatures/Temperatures.ts
const n: number = parseInt(readline());
var inputs: string[] = readline().split(' ');
let res = 0;
for (let i = 0; i < n; i++) {
    const t: number = parseInt(inputs[i]);
    i == 0 ? res = t : null;
    Math.abs(t) < Math.abs(res) ? res = t : null;
    Math.abs(t) == Math.abs(res) ? res = Math.max(res, t) : null;
}
console.log(res);

File: Easy/Temperatures/Temperatures.py
import sys
import math

n = int(input())
res = None
for i in input().split():
    t = int(i)
    res = (t if res == None or abs(t) < abs(res) else res)
    res = (max(t, res) if abs(t) == abs(res) else res)
print(res if n > 0 else 0)
